lib/processors/lib/utils.py

```diff
@@ -11,7 +12,11 @@ def get(self):
             userid = self.get_argument('userid')
             privatekey_pem = self.get_argument('privatekey', None)
             publickey_pem = self.get_argument('publickey', None)
-            user = User(userid, publickey_pem=publickey_pem, privatekey_pem=privatekey_pem)
+            user_blob = get_user(userid)
```

Comment:
I thought we agreed not to have gulper connect to the DB... whatever processor needs this information can just store it on scrape since we provide that data to the `User` object.  Also, this statement will fail since `get_user` is a coroutine, so if this code is tested it seems to me nothing uses this data right now anyways.